src/experiments/rl_envs.jl

```diff
@@ -553,10 +540,33 @@ function RLCore.Experiment(
         ),
     )
 
-
-    hook = TotalBatchRewardPerEpisode(N_ENV)
-    stop_condition = StopAfterStep(100000)
-    Experiment(agent, env, stop_condition, hook, "# A2C with CartPole")
+    stop_condition = StopAfterStep(10_000)
+    total_reward_per_episode = TotalBatchRewardPerEpisode(N_ENV)
+    time_per_step = TimePerStep()
+    hook = ComposedHook(
+        total_reward_per_episode,
+        time_per_step,
+        DoEveryNStep() do t, agent, env
+            with_logger(lg) do
+                @info "training" actor_loss = agent.policy.learner.actor_loss
+                @info "training" critic_loss = agent.policy.learner.critic_loss
+                @info "training" entropy_loss = agent.policy.learner.entropy_loss
+                @info "training" loss = agent.policy.learner.loss
+                @info "training" reward = total_reward_per_episode.reward[end]
```

Comment:
Either merge them into a single `@info` expression or add ` log_step_increment = 0` at the end of `@info` except the first one. See more details here: https://philipvinc.github.io/TensorBoardLogger.jl/dev/#Basic-Usage

> Every TBLogger has an internal counter to store the current step, which is initially set to 1. All the data logged with the same @log call will be logged with the same step, and then it will increment the internal counter by 1.

> If you want to increase the counter by a different amount, or prevent it from increasing, you can log the additional message log_step_increment=N. The default behaviour corresponds to N=1. If you set N=0 the internal counter will not be modified. The defaul behaviour for logger lg can be changed executing set_step_increment!(lg, N).

Reply:
Thanks for your help.
I was wondering why the step counts were increased.